po/nl.po

```diff
@@ -0,0 +1,255 @@
+# FIDEI GETTEXT TRANSLATION TEMPLATE.
+# Copyright (C) 2022 Florian "sp1rit" <sp1rit@national.shitposting.agency>
+# This file is distributed under the same license as Fidei.
+# Florian "sp1rit" <sp1rit@national.shitposting.agency>, 2022.
```

Comment:
mind dropping/replacing the the first line as this is not a template anymore and update the copyright + author to 2024 and yourself?

Reply:
Oh, seems like Poedit didn't replace that for some reason. :/ Will do that.

Reply:
please update the copyright year and change the author to yourself.

also see the small-caps regex https://github.com/sp1ritCS/fidei/pull/2#pullrequestreview-2425052698

Reply:
Hmm, you point out stuff that no one ever points out. Good catch! 🙂

What do you mean with the small-caps regex?

Reply:
In classical typesetting, "Lord" is generally set in small caps (see [this](https://www.gotquestions.org/LORD-GOD-Lord-God.html)).

While Unicode technically has small caps codepoints, they generally aren't used and replaced with an all caps "LORD".

Fidei matches those all-caps occurrences based on a regex for each language and converts it back into small caps, as it looks better.

While users can add languages / change the expressions themselves in the preferences, I'd like to have the defaults reflect the supported languages.

Reply:
`data/arpa.sp1rit.Fidei.gschema.xml` has the linked `small-caps` key, whose format maps a two letter language code onto a regular expression. The default currently contains expression for `en`, `de` and `la` and you'd add yours to that default map.

If you have issues with writing/validating regular expressions, you can try it first on https://regex101.com

Reply:
Thanks for the instructions. I've done that now.